deployment/kubernetes/yaml/cdcservice.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ftgo-cdc-service
+  namespace: default
+spec:
+  ports:
+  - port: 8080
+    targetPort: 8080
+  selector:
+    svc: ftgo-cdc-service
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ftgo-cdc-service
+  labels:
+    application: ftgo
+spec:
+  selector:
+    matchLabels:
+      svc: ftgo-cdc-service
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 0
+  template:
+    metadata:
+      labels:
+        svc: ftgo-cdc-service
+    spec:
+      containers:
+      - name: ftgo-cdc-service
+        image: eventuateio/eventuate-cdc-service:0.4.0.RELEASE
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8080
+          name: httpport
+        command: ["bash", "-c", "java -Dsun.net.inetaddr.ttl=30 -jar *.jar" ]
+        env:
+          - name: SPRING_DATASOURCE_URL

```

Comment:
Please set up the `env:` based on the environment variables defined in the `eventuate-tram-examples-customers-and-orders/docker-compose-mysql-binlog.yml` file

Reply:
updated